src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -153,6 +153,13 @@ private bool TryGetTextForSymbol(SyntaxToken token, SemanticModel semanticModel,
                 symbol = ((ILocalSymbol)symbol).Type;
             }
 
+            // Range variable: use the type
+            if (symbol is IRangeVariableSymbol)
+            {
+                var info = semanticModel.GetTypeInfo(token.Parent, CancellationToken.None);
```

Comment:
Why .None?   We should have a cancellation token somewhere that we can pass through to this, right?